### PR TITLE
Custom costs base

### PR DESCRIPTION
**Pull request by roopehub**
- made public attributes private and created getter from them for following better java practices 
- made attributes and getters for base travel times (per edge) and addition travel 'cost seconds' for being able to save the base travel times per osmid/edge/way and the addition cost/'cost seconds' which are added to each edge 
- added test cases

### CODE REVIEW

File: src/main/java/com/conveyal/r5/rastercost/CustomCostField.java [hunk 1]
```diff
@@ -122,8 +145,11 @@ public int additionalTraversalTimeSeconds (EdgeStore.Edge currentEdge, int baseT
         if (Double.isNaN(additionalCostSeconds)) {
             throw new CustomCostFieldException("Custom cost addition result is NaN for osmId:  " + currentEdge.getOSMID());
         }
+        int roudedAdditionalCostSeconds = (int) Math.round(additionalCostSeconds);
```
Comment (christophfink) [inline src/main/java/com/conveyal/r5/rastercost/CustomCostField.java]: the variable name has a typo I assume :)

File: src/main/java/com/conveyal/r5/rastercost/CustomCostField.java [hunk 2]
```diff
@@ -122,8 +145,11 @@ public int additionalTraversalTimeSeconds (EdgeStore.Edge currentEdge, int baseT
...
+        // save the custom cost addition costs
+        customCostAdditionalTraveltimes.put(keyOsmId, roudedAdditionalCostSeconds);
         // value is rounded and casted to int for seconds
-        return (int) Math.round(additionalCostSeconds);
+        return roudedAdditionalCostSeconds;
```
Comment (christophfink) [inline src/main/java/com/conveyal/r5/rastercost/CustomCostField.java]: same

**Review by christophfink — APPROVED** [review]
Except for the very small thing (a typo in a variable name), all looks good to me (but then again, I’m not a Java expert)

File: src/main/java/com/conveyal/r5/rastercost/CustomCostField.java (hunk 1 above)
Comment (roopehub) [reply src/main/java/com/conveyal/r5/rastercost/CustomCostField.java]: Good spotting, will fix all of this typos!

**Comment by roopehub** [comment]
Good! I'm either not a Java profi, but will merge this now. Edits and modifications in a new PR if needed.

**Pull request merged**